packages/server/utils/AzureDevOpsServerManager.ts

```diff
@@ -233,6 +256,24 @@ class AzureDevOpsServerManager {
     return json
   }
 
+  private readonly patch = async <T>(url: string, payload: any) => {
+    const patchHeaders = this.headers
+    patchHeaders['Content-Type'] = 'application/json-patch+json'
```

Comment:
-1 Since you're only assigning a reference in line 260 this will modify `this.headers` which is most likely not what you want. Instead it would be easier if you would modify the next lines
```
    const res = await this.fetchWithTimeout(url, {
      ...
      headers: {
        ...this.headers,
        ['Content-Type']: 'application/json-patch+json'
      }
      ...
    })

Reply:
I made the change.  I just want confirmation from @wdfox that this was the intent.  I will mark resolved when confirmed.